docs/0.5/howto/removing_a_splinter_circuit.md

```diff
@@ -0,0 +1,366 @@
+# Removing a Splinter Circuit
+
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+This guide explains alternative approaches to delete a circuit, disbanding and
+abandoning. Both of these processes result in removing the circuit’s
+networking, allowing for the circuit to be purged in either case. Disbanding a
+circuit begins with a node’s administrator proposing to remove a circuit’s
+networking functionality. Then, all members are able to vote before the circuit
+is disbanded. Once the circuit is disbanded, the circuit will no longer be
+usable to send messages and the circuit is only available to each node locally.
+On the other hand, a node’s administrator is able to abandon a circuit,
+breaking the circuit’s networking ability for all other circuit members. Both
+abandoning and disbanding a circuit result in removing a circuit’s networking
+capability. Though these options use different processes, both make the circuit
+only available locally and able to be purged. A node administrator may purge a
+circuit once it has been disbanded or abandoned in order to delete a circuit
+from state and remove any internal service data.
+
+## Removing a Splinter Circuit
+
+Splinter administrators have the option to disband or abandon a circuit, the
+first part of deleting a circuit. Whether a circuit is disbanded or abandoned,
+the networking functionality is disabled for the requesting node. If a circuit
+is abandoned, the other circuit members do not disable the circuit’s networking
+and may be able to use the circuit without the abandoning node. However, these
+options involve different processes and it is important to note these
+differences before using either operation.
+
+An individual node’s administrator may choose to abandon a circuit,
```

Comment:
There is a fair amount of duplication in the first few paragraphs.

Reply:
Definitely. Tried to shorten that first paragraph, providing a little bit more details in the following paragraph below the first header.